lib/core/src/send_swap.rs

```diff
@@ -93,14 +94,35 @@ impl SendSwapHandler {
                 Ok(())
             }
 
-            // Boltz has detected the lockup in the mempool, we can speed up
-            // the claim by doing so cooperatively
+            // Boltz has detected the lockup in the mempool. If the swap is not to be batched
+            // we can speed up the claim by doing so cooperatively.
             SubSwapStates::TransactionClaimPending => {
                 if swap.metadata.is_local {
-                    self.cooperate_claim(&swap).await.map_err(|e| {
-                        error!("Could not cooperate Send Swap {id} claim: {e}");
-                        anyhow!("Could not post claim details. Err: {e:?}")
-                    })?;
+                    let pair = swap.get_boltz_pair()?;
+                    if swap.receiver_amount_sat < pair.limits.minimal {
```

Comment:
Sorry for the addition here but I think that we shouldn't count on the minimal limits on whether to cooperate with the claim or not since it is up to the swapper on whether to cooperate or not.
I think we should:
1. Try to cooperative claim anyway and ignore the failure.
2 If we failed to cooperate (still don't have the preimage) use the other endpoint to get it.

Reply:
Done [4caafc3](https://github.com/breez/breez-sdk-liquid/pull/863/commits/4caafc32fd5133b8cf6b306632679a70a69d055b)